robowflex_library/scripts/resources_visualization.cpp

```diff
@@ -38,10 +38,10 @@ const std::string ur5[4] = {
 
 // Franka Emika Panda robot with hand
 const std::string panda[4] = {
-    "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro",    // urdf
-    "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro",  // srdf
-    "package://robowflex_resources/panda/config/joint_limits.yaml",          // joint limits
-    "package://robowflex_resources/panda/config/kinematics.yaml"             // kinematics
+    "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro",     // urdf
```

Comment:
I think the new files are named ``package://robowflex_resources/panda/panda_description/urdf/panda.urdf `` etc.

Reply:
yes. wrong commit. sorry about that.
Thanks for catching it!